src/components/common/Extensions/UserHub/partials/ReputationTab/partials/Balance.tsx

```diff
@@ -84,7 +85,7 @@ const Balance: FC<BalanceProps> = ({ nativeToken, wallet, onTabChange }) => {
             className={reputationTabClasses.numeral}
             value={tokenBalanceData?.balance ?? 0}
             decimals={tokenDecimals}
-            suffix={` ${nativeToken?.symbol || 'CLNY'}`}
+            suffix={` ${truncateTokenSymbol(nativeToken?.symbol) || 'CLNY'}`}
```

Comment:
Why does this have CLNY as a fall back, it really shouldn't be here. @jakubcolony any ideas